docs/content/preview/admin/yb-ts-cli.md

```diff
@@ -79,6 +80,19 @@ yb-ts-cli [ --server_address=<host>:<port> ] is_server_ready
 
 * *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
 
+##### clear_server_metacache
+
+Clears all metacaches that are stored on a specified server. Works on both YB-Master (port 9100) and YB-TServer (port 7100) processes.
+
+Metacache is a YBClient cache on a Tablet/Master server which is used to find out which Tablet Server hosts which tablet. Because this cache can be stale in some cases, this command is introduced for users to clear the metacache on a particular YB-TServer or YB-Master.
```

Comment:
What is YBClient? @mdbridge

Reply:
YBClient is a client class living inside a T-Server, it is what the users talk to when they connect to a tablet server. It has many usage, but as an example, in our case, when a T-Server received a write request from the user, the YBClient will use its meta-cache to figure out which tablet this request needs to write to, and if the T-Server itself does not host that tablet, it will go talk to master to find out what tablet it should issue the request to, cache this information in the meta-cache and retry the RPC request to the correct T-Server. @mdbridge please correct me or add more information if I am missing anything. @ddhodge Do you think I should add more detailed info for this command?

Reply:
I agreed that YBClient  is probably a layer of detail too far.

Maybe something closer to:

Tablet servers and masters use MetaCache's to cache information about which tablet server hosts which tablet.  Because these caches could become stale in some cases, you may want to use this command to clear the MetaCaches on a particular tablet server or master.

Reply:
I wouldn't add any description of ybclient or more detail, no. ybclient doesn't appear (well, hardly) in the docs elsewhere, so that's why I was asking. mbridge's suggestion seems good.

Reply:
Just committed ro remove YBClient reference, please check if this satisfies the requireent

Reply:
LGTM

Reply:
@SeanSong25 what versions does this apply to? Only 2.21?

Reply:
Yes, I've only backported it to 2024.1, and not anything before that

Reply:
can I squash and merge this?

Reply:
If by that you mean it only applies to preview, then yes, please merge, and thanks